src/common/extension.ts

```diff
@@ -0,0 +1,28 @@
+// ==LICENSE-BEGIN==
+// Copyright 2017 European Digital Reading Lab. All rights reserved.
+// Licensed to the Readium Foundation under one or more contributor license agreements.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file exposed on Github (readium) in the project repository.
+// ==LICENSE-END=
+
+import { ObjectKeys, ObjectValues } from "readium-desktop/utils/object-keys-values";
+
+export const acceptedExtensionObject = {
+    lcpLicence: ".lcpl",
+    epub2: ".epub",
```

Comment:
The `epub2` key should be `epub`, as in fact this covers EPUB v2, v3, etc.

Explanation: the `.epub3` extension is a non-standard oddity implemented by https://standardebooks.org :)

Reply:
done :)